content/support/missing-history/index.md

```diff
@@ -15,16 +15,7 @@ If you were working with an unregistered account, the History could really be no
 
 # Someone shared a history with me, where do I find it?
 
-When logged into your account, these are located under **User > Histories Shared with Me**.
-
-**Working with _Histories Shared with Me_**
-
-* Make copies to import the History into your account where they can be worked with.
-* Unshare Histories that you no longer want shared with you or that you have already made a copy of.
-* Shared Histories (when copied into your account or not) do count in portion toward your [total account data quota usage](/support/account-quotas/) at Galaxy Main http://usegalaxy.org and most other [public Galaxy servers](/use).
-* Copies of Histories from others count in full toward your total account data quota. Data copied from others is the same as all of your other active/deleted but not permemenently deleted/purged data.
-* More details about how data shared with you, or that you shared with others, functions with respect to account quota usage [are included in this FAQ](/support/account-quotas/#find-histories-that-have-been-shared-with-you-and-unshare-those-not-needed).
-* Full details about data privacy features and settings [are included in this FAQ](/learn/privacy-features/).
```

Comment:
Do we have something on data privacy features and settings in the GTN? (I can't remember!)

Reply:
[this](https://github.com/galaxyproject/training-material/pull/3357/)?